R/raw-data.R

```diff
@@ -0,0 +1,72 @@
+#' @importFrom dplyr mutate
+#' @importFrom purrr map_dfr
+#' @importFrom readr read_csv
+#' @importFrom glue glue
+#' @importFrom vroom problems
+
+
+# The package will store releases internally, and not in raw data
+# Until that happens, this will duplicate the bundle code
+
+#' @title Get available raw bundles
+#'
+#' @description If a bundle has multiple names, then the id will be duplicated across rows
+#'
+#' @return dataframe that contains a "concept_name" and a "domain" column for each available concept
+raw_bundles <- function() {
+  raw_dir <- get_raw_dir()
+  directories <- dir(raw_dir, full.names = TRUE)
+  bundle_name_paths <- file.path(directories, "bundle_names.csv")
+
+  purrr::map_dfr(bundle_name_paths, parse_bundle_names)
+}
+
+
+get_raw_dir <- function(..., version = "latest") {
+  if (version != "latest") warning("Versioning not yet implemented, using version = 'latest'")
+
+  file_path <- system.file("data-raw", ..., package = "omopbundles")
+
+  if (!file.exists(file_path)) {
+    path <- paste(..., sep = "/")
```

Comment:
```suggestion
    path <- file.path(...)
```

To make it OS-agnostic (i.e. deal with Windows...) better to use `file.path()`, which will put the appropriate separator based on the OS.

Reply:
Ah wait, never mind, didn't realise this was just for the error messager